### ISSUE TITLE
custom-lovelace: bignumber-card.js throttles cpu to 100%

### ISSUE THREAD

**Issue by psecto** [ciotlosm/custom-lovelace#204]
Hi,

I love your script, thank you for providing such knowledge to the community. I am just starting up with home assistant and I have stumbled upon your bignumber-card example. The cards work fine and I love the customisation options, however I cannot really use them in a 24/7 display as it keeps the cpu to 100% all time. 

Is it possible that the script enters in a loop somehow? I am not a coder myself, being at most, a merely code tinker.

All the best, and thanks again for this free knowledge 👍

<img width="1680" alt="screen shot 2018-09-28 at 22 50 34" src="https://user-images.githubusercontent.com/20581732/46232915-22f4f600-c371-11e8-9568-f6eab4646d64.png">

**Comment by psecto** [comment]
Nvm, please close this thread. Safari seems to have a problem with it as it runs it inside a container. Chrome works just fine. 

Thanks again for this awesome script.

**Comment by ciotlosm** [comment]
@psecto Did you find the root cause of your 100% CPU? Might help other users if you at mentioned also the solution to your problem.

**Comment by psecto** [comment]
I had the impression that I've submitted already my last comment :)

Here is what I found out: 

- It appears that this only happens in Safari running on MacOS. Javascript code when running in Safari, appears to run into its own app outside of the web browser itself. I don't think it's your script since I also found other javascript intensive pages which are doing the same. 

Running Chrome on Mac OS poses no issues.

Best,
psecto
